pyaerocom/colocation/colocated_data.py

```diff
@@ -899,8 +892,12 @@ def check_set_countries(self, inplace=True, assign_to_dim=None):
         arr = arr.assign_coords(
             country=(assign_to_dim, countries), country_code=(assign_to_dim, codes)
         )
-        coldata.data = arr
-        return coldata
+        if inplace:
+            self.data = arr
+            return self
+        else:
+            coldata = arr
```

Comment:
This looks wrong? coldata is in most cases something like self, so I would expect it to be `coldata.data = arr` for symmetry.